public/javascripts/running.js

```diff
@@ -168,65 +148,60 @@ function enableInteractive() {
 }
 
 function disableInteractive() {
-    window.hide("stop_button");
+    $("#stop_button").hide();
     sendCommand("disable_interactive_mode");
 }
 
 function sendCommand(command) {
-    var wid = window.testStatus.workerid;
+    var wid = testStatus.workerid;
     if (wid == null) return false;
-    new Ajax.Request("/api/v1/workers/" + wid + "/commands", {
-        method: "post",
-        parameters: { command: command },
-        onSuccess: function(resp) {
-            window.setTimeout("updateStatus()", 0);
-        }});
+    $.ajax({url: "/api/v1/workers/" + wid + "/commands",
```

Comment:
it's easier there as it's static per test - here we have to wait for the workerid to be passed by ajax. But I'll manage :)